pkg/neotest/doc.go

```diff
@@ -21,5 +21,11 @@ them in the same package with the smart contract iself can lead to unxpected
 results if smart contract has any init() functions. If that's the case they
 will be compiled into the testing binary even when using package_test and their
 execution can affect tests. See https://github.com/nspcc-dev/neo-go/issues/3120 for details.
+
+Test coverage for contracts works as expected and is automatically enabled when
```

Comment:
s/Test/Neotest?

Reply:
I explicitly said `for contracts`, `neotest coverage` sounds a bit weird.